### PR TITLE
Improved documentation site. Adding a copy button for code blocks 

### PR DESCRIPTION
**Pull request by TOsmanov**
This PR added `copy` button for code blocks by enabling [prism](https://prismjs.com/index.html). 
`Copy` button enabling for all versions.
`Copy` button displayed if language specified.
Updated README.md.

Preview:
https://tosmanov.github.io/cvat/docs/administration/basics/installation/
https://tosmanov.github.io/cvat/docs/manual/advanced/serverless-tutorial/

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by azhavoro** [comment]
@TOsmanov Hi, is it possible to make the `Copy` button visible without hovering over a code block?

**Comment by azhavoro** [comment]
Thanks, LGTM, but please fix linter issues

File: site/assets/scss/_custom.scss [hunk 1]
```diff
@@ -2,6 +2,32 @@
 //
 // SPDX-License-Identifier: MIT
 
+/* Code blocks */
+
+div.code-toolbar .toolbar {
```
Comment (nmanovic) [inline site/assets/scss/_custom.scss:7]: Why do we need these styles? Can we use default?

File: site/content/en/docs/administration/basics/installation.md [hunk 2]
```diff
@@ -444,14 +444,14 @@ If you stay in China, for installation you need to override the following source
...
-  ```
+  ```bash
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal main restricted universe multiverse
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-updates main restricted universe multiverse
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-backports main restricted universe multiverse
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-security main restricted universe multiverse
   ```
   Or source packages:
-  ```
+  ```bash
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/installation.md:454]: I see several ways to mark similar code: bash, sh, console. Could you please choose one?

File: site/assets/scss/_custom.scss (hunk 1 above)
Comment (TOsmanov) [reply site/assets/scss/_custom.scss:7]: Styles are needed to correct the location and transparency of the `copy` button, make the background color of code blocks the same as without specifying the language, fixes the display of code blocks having a `console` language in the desktop and mobile versions

no styles:
image

with styles:
image

File: site/content/en/docs/administration/basics/installation.md (hunk 2 above)
Comment (nmanovic) [reply site/content/en/docs/administration/basics/installation.md]: @TOsmanov , could you please answer?

Comment (TOsmanov) [reply site/content/en/docs/administration/basics/installation.md]: in the `installation.md` file I replaced `bash` with  `sh`

Comment (TOsmanov) [reply site/content/en/docs/administration/basics/installation.md:454]: I replaced the language `console` and `sh` with `bash` in the latest version of the documentation as `bash` looks better, but css for other languages should remain in the `_custom.css` as it is used in older versions of the documentation

**Review by nmanovic — APPROVED** [review]
LGTM

**Pull request merged**